test/func/tests/plugins/tests-menu-bar-plugin.testplane.js

```diff
@@ -12,6 +12,9 @@ describe('Test menu bar plugin', function() {
         await browser.$(selector).waitForDisplayed();
         await browser.$(selector).click();
         await browser.$(menuSelector).waitForDisplayed();
+
+        // Pause prevents flaky screenshots due to dropdown shadow rendering. No, screenshot delay doesn't help.
+        await browser.pause(1000);
```

Comment:
Does it need to be that high?

Reply:
Unfortunately, yes =(
In my tests, 500 wasn't enough. It's very annoying and weird, but I don't want to spend time on this right now